tools/report-converter/tests/unit/util/test_trim_path_prefix.py

```diff
@@ -57,3 +57,12 @@ def test_prefix_in_filename(self):
                          trim_path_prefixes(test_path, ['/a/b/common']))
         self.assertEqual('common.txt',
                          trim_path_prefixes(test_path, ['/a/b/']))
+
+    def test_prefix_blob(self):
+        test_path = '/a/b/common.txt'
+        self.assertEqual('common.txt', trim_path_prefixes(test_path, ['/*/b']))
+        self.assertEqual('common.txt', trim_path_prefixes(test_path, ['/a/*']))
+        self.assertEqual('b/common.txt',
+                         trim_path_prefixes(test_path, ['/*/']))
```

Comment:
This test case doesn't pass, because `trim_path_prefixes()` function returns `common.txt` on these inputs. The test fails, but it is not presented probably due to another style checker:
```
codechecker_report_converter/util.py:91:17: E125 continuation line with same indent as next logical line
```
By the way I think it is correct that this function returns 'common.txt' on these parameters, because we should mimic shell behavior where `*` substitutes multiple characters, including `/`.

Please, extend the documentation of `trim_path_prefixes()` with the behavior of joker characters (`?`. `*`), and also mention this in the command-line documentations:
https://github.com/Ericsson/codechecker/blob/6499dfdca5e13ca356081d368fe4cd38a93f5d19/docs/analyzer/user_guide.md?plain=1#L1626
https://github.com/Ericsson/codechecker/blob/6499dfdca5e13ca356081d368fe4cd38a93f5d19/docs/web/user_guide.md?plain=1#L457
https://github.com/Ericsson/codechecker/blob/6499dfdca5e13ca356081d368fe4cd38a93f5d19/analyzer/codechecker_analyzer/cmd/check.py#L707
https://github.com/Ericsson/codechecker/blob/6499dfdca5e13ca356081d368fe4cd38a93f5d19/analyzer/codechecker_analyzer/cmd/parse.py#L182
https://github.com/Ericsson/codechecker/blob/6499dfdca5e13ca356081d368fe4cd38a93f5d19/web/client/codechecker_client/cmd/store.py#L240